src/OpenTelemetry.Instrumentation.StackExchangeRedis/StackExchangeRedisInstrumentationOptions.cs

```diff
@@ -22,6 +22,19 @@ public class StackExchangeRedisInstrumentationOptions
     /// </summary>
     public bool SetVerboseDatabaseStatements { get; set; }
 
+    /// <summary>
+    /// Gets or sets the callback method allowing to filter out particular <see cref="IProfiledCommand"/>.
+    /// </summary>
+    /// <remarks>
+    /// The filter callback receives the <see cref="IProfiledCommand"/> for the
+    /// processed profiled command and should return a boolean.
```

Comment:
Other than throwing, it's not possible to not return a boolean so "should" doesn't sound right:

```suggestion
    /// processed profiled command and returns a boolean indicating whether it should be filtered out.
```